Scripts/Mac/BuildInstaller.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
+  CONFIGURATION=Debug
+fi
+
+PACKAGEVERSION=$2
+if [ -z $PACKAGEVERSION ]; then
+  PACKAGEVERSION="0.0.0.1"
+fi
+
+INSTALLROOT=$VFS_OUTPUTDIR/"InstallerRoot"
+VFSFORGITDESTINATION="usr/local/vfsforgit"
+KEXTDESTINATION="$VFSFORGITDESTINATION"
+INSTALLERPACKAGENAME="VFSForGit.$PACKAGEVERSION"
+INSTALLERPACKAGEID="com.vfsforgit.pkg"
+
+function CheckBuildIsAvailable()
+{
+    if [ ! -d $VFS_OUTPUTDIR ] || [ ! -d $VFS_PUBLISHDIR ]; then
```

Comment:
Do we need to be dependent on publishing? Can we set it up to trigger the publish if it hasn't happened yet? What if the build happened, the publish hasn't, and this script is now packaging up stale binaries?

Reply:
This script is now being called from GVFS.Installer.Mac.csproj. The installer creation is hooked on the "Publish" target. Please take a look at the new code and let me know your thoughts.